sdk/KoreBuild/msbuild/KoreBuild.RepoTasks.Sdk/Sdk/Sdk.props

```diff
@@ -10,7 +10,8 @@
       The dependency versions here don't matter as long as the version is binary compatible with the version of MSBuild and NuGet
       already embedded in Microsoft.NET.Sdk. These are essentially reference assemblies.
     -->
-    <MSBuildPackagesVersion>15.3.0-preview-000388-01</MSBuildPackagesVersion>
+    <MicrosoftBuildVersion>15.3.0-*</MicrosoftBuildVersion>
```

Comment:
Let's leave this one hardcoded instead of floating. The version doesn't really matter here as they are just reference assemblies.